Source/GUI/MainWindow.cpp

```diff
@@ -74,6 +73,10 @@ void MainWindow::makeMenus()
 
   m_actMemoryViewer = new QAction(tr("&Memory Viewer"), this);
   m_actCopyMemory = new QAction(tr("&Copy Memory Range"), this);
+  m_actScanner = new QAction(tr("&Scanner"), this);
```

Comment:
Whatever you decide in the Label under View lets add in parens the key.

Such that that item would say:
`Scanner (F3)`